build/update-attribution-files/make_attribution.sh

```diff
@@ -50,7 +49,32 @@ RELEASE_FOLDER=$(find $PROJECT_ROOT -type d -name "1-*")
 if [ -z "${RELEASE_FOLDER}" ]; then
     build::attribution::generate
 else
+    LAST_GIT_TAG=""
+    LAST_RELEASE_BRANCH=""
     for release in $(cat $MAKE_ROOT/release/SUPPORTED_RELEASE_BRANCHES) ; do
-        build::attribution::generate $release
+        export RELEASE_BRANCH="$release"
+
+        GIT_TAG="$(cat $PROJECT_ROOT/$release/GIT_TAG)"
+        if [ "$GIT_TAG" != "$LAST_GIT_TAG" ]; then
+            # clean before regenerating to ensure there are no intermediate files left around
+            make -C $PROJECT_ROOT clean clean-go-cache
+            build::attribution::generate $release
+        else
+            # if the git_tags match across release branches, save the output state to avoid
+            # rebuilding/regenerating
+            if [[ $TARGET == *"checksums"* ]]; then
+                mkdir -p $PROJECT_ROOT/_output/$release
+                cp -rf $PROJECT_ROOT/_output/$LAST_RELEASE_BRANCH/bin $PROJECT_ROOT/_output/$release
+            fi
+            if [[ $TARGET == *"attribution"* ]]; then
+                mkdir -p $PROJECT_ROOT/_output/$release
+                cp -rf $PROJECT_ROOT/_output/$LAST_RELEASE_BRANCH/{attribution,LICENSES} $PROJECT_ROOT/_output/$release
+            fi
+            build::attribution::generate $release
```

Comment:
Do we need to run this again, if the git tag hasn't changed?

Reply:
We could do it another way, but the way this implements it, yes.  The intermediate files (LICESES/attribution/bin) are copied between the release branch output folders so the target still needs to run to generate the checksum file/copy attribution from _output to actual release_branch folder.  They ending up being super quick.